python/sgl_jax/srt/server_args.py

```diff
@@ -753,9 +730,9 @@ def add_cli_args(parser: argparse.ArgumentParser):
             help="Set the list of batch sizes buckets for jax jit",
         )
         parser.add_argument(
-            "--disable-jax-precompile",
+            "--disable-precompile",
```

Comment:
Should the documentation be updated accordingly to reflect these changes in startup parameter naming?